docs/bigquery/snippets.py

```diff
@@ -1810,6 +1952,130 @@ def test_client_query_destination_table_cmek(client, to_delete):
     # [END bigquery_query_destination_table_cmek]
 
 
+def test_client_query_relax_column(client, to_delete):
+    dataset_id = 'query_relax_column_{}'.format(_millis())
+    dataset_ref = client.dataset(dataset_id)
+    dataset = bigquery.Dataset(dataset_ref)
+    dataset.location = 'US'
+    dataset = client.create_dataset(dataset)
+    to_delete.append(dataset)
+
+    dest_table_ref = dataset_ref.table('my_table')
+    schema = [
+        bigquery.SchemaField('full_name', 'STRING', mode='REQUIRED'),
+        bigquery.SchemaField('age', 'INTEGER', mode='REQUIRED'),
+    ]
+    dest_table = client.create_table(
+        bigquery.Table(dest_table_ref, schema=schema))
```

Comment:
This variable is unused, which is what caused the [CI failure for `lint`](https://circleci.com/gh/GoogleCloudPlatform/google-cloud-python/6764).